src/theme/defaultTheme.js

```diff
@@ -32,6 +32,11 @@ const colors = {
 };
 
 const components = {
+  globalStyleOverride: css`
+    & button {
+      border: solid 1px ${colors.borderGrey};
+    }
+  `,
```

Comment:
not sure about the intent here.. are we trying to globally override other styles that we've used in child components? if that's the case it seems more appropriate to fix the things that we're trying to override, rather than overriding them. but maybe I'm missing the point? thoughts?

Reply:
it's there to override browser defaults, which oddly enough look different in Chrome on windows vs Chrome on Mac. If any styling issue arises I think it can be addressed on a case-by-case basis

Reply:
oh gotcha, yeah sounds good